packages/block-library/src/list/save.js

```diff
@@ -1,15 +1,18 @@
 /**
  * WordPress dependencies
  */
-import { RichText, useBlockProps } from '@wordpress/block-editor';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { ordered, values, type, reversed, start } = attributes;
+	const { ordered, reversed, start } = attributes;
 	const TagName = ordered ? 'ol' : 'ul';
-
 	return (
-		<TagName { ...useBlockProps.save( { type, reversed, start } ) }>
```

Comment:
This PR removes the `type` attribute (both from `edit` and `save`), which means that we'll no longer allow changing the [type](https://www.w3schools.com/tags/att_ol_type.asp) of the OL list. Is this intentional?

Reply:
Hm, I didn't write this, so I'm not sure. Looks like this is wrong. Also this has to be passed through `useBlockProps.save()`.